src/tasks/pair-liquidity-info-history-importer/pair-liquidity-info-history-importer-v2.service.spec.ts

```diff
@@ -0,0 +1,527 @@
+  data: '',
+  event_hash: '6O232NLB36RGK54HEJPVDFJVCSIVFV29KPORC07CSSDARM7LV4L0====', // Sync
+  height: '10001',
+  log_idx: '1',
+};
+const contractLog2 = {
+  args: ['123', '100', '100'],
+  block_hash: 'mh_hash1',
+  block_time: '1000000000001',
+  call_tx_hash: 'th_hash1',
+  call_txi: '10000001',
+  data: '',
+  event_hash: 'L2BEDU7I5T8OSEUPB61900P8FJR637OE4MC4A9875C390RMQHSN0====', // PairMint
+  height: '10001',
+  log_idx: '2',
+};
+const contractLog3 = {
+  args: ['200', '200'],
+  block_hash: 'mh_hash2',
+  block_time: '2000000000002',
+  call_tx_hash: 'th_hash2',
+  call_txi: '20000002',
+  data: '',
+  event_hash: '6O232NLB36RGK54HEJPVDFJVCSIVFV29KPORC07CSSDARM7LV4L0====', // Sync
+  height: '20002',
+  log_idx: '1',
+};
+const contractLog4 = {
+  args: ['201', '199'],
+  block_hash: 'mh_hash3',
+  block_time: '3000000000003',
+  call_tx_hash: 'th_hash3',
+  call_txi: '30000003',
+  data: '',
+  event_hash: '6O232NLB36RGK54HEJPVDFJVCSIVFV29KPORC07CSSDARM7LV4L0====', // Sync
+  height: '30003',
+  log_idx: '1',
+};
+const contractLog5 = {
+  args: [],
+  block_hash: 'mh_hash3',
+  block_time: '3000000000003',
+  call_tx_hash: 'th_hash3',
+  call_txi: '30000003',
+  data: '1|0|0|1',
+  event_hash: 'K39AB2I57LEUOUQ04LTEOMSJPJC3G9VGFRKVNJ5QLRMVCMDOPIMG====', // SwapTokens
+  height: '30003',
+  log_idx: '2',
+};
+const contractLog6 = {
+  args: ['100', '100'],
+  block_hash: 'mh_hash3',
+  block_time: '3000000000003',
+  call_tx_hash: 'th_hash4',
+  call_txi: '40000004',
+  data: '',
+  event_hash: '6O232NLB36RGK54HEJPVDFJVCSIVFV29KPORC07CSSDARM7LV4L0====', // Sync
+  height: '30003',
+  log_idx: '1',
+};
+const contractLog7 = {
+  args: [],
+  block_hash: 'mh_hash3',
+  block_time: '3000000000003',
+  call_tx_hash: 'th_hash4',
+  call_txi: '40000004',
+  data: '101|99',
+  event_hash: 'OIS2ALGSJ03MTP2BR5RBFL1GOUGESRVPGE58LGM0MVG9K3VAFKUG====', // PairBurn
+  height: '30003',
+  log_idx: '2',
+};
+const contractLog8 = {
+  args: [],
+  block_hash: 'mh_hash3',
+  block_time: '3000000000003',
+  call_tx_hash: 'th_hash4',
+  call_txi: '40000004',
+  data: '',
+  event_hash: 'non_relevant_event_hash', // Something else
+  height: '30003',
+  log_idx: '3',
+};
+
+const mockMdwClientService = {
+  getContract: jest.fn(),
+  getMicroBlock: jest.fn(),
+  getContractLogsUntilCondition: jest.fn(),
+};
+
+const mockPairDb = { getAll: jest.fn() };
+
+const mockPairLiquidityInfoHistoryDb = {
+  getLastlySyncedLogByPairId: jest.fn(),
+  upsert: jest.fn(),
+};
+
+const mockPairLiquidityInfoHistoryErrorDb = {
+  getErrorWithinHours: jest.fn(),
+  upsert: jest.fn(),
+};
+
+describe('PairLiquidityInfoHistoryImporterV2Service', () => {
+  let service: PairLiquidityInfoHistoryImporterV2Service;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [
+        PairLiquidityInfoHistoryImporterV2Service,
+        { provide: MdwHttpClientService, useValue: mockMdwClientService },
```

Comment:
```suggestion
        { provide: MdwHttpClientService, useValue: mockMdwClient },
```
keep the naming consistent, service everywhere or nowhere :)